### ISSUE TITLE
Other Logitech keyboards?

### ISSUE THREAD

**Issue by allantokuda** [milgner/k290-fnkeyctl#12]
I have a Logitech K760 Bluetooth keyboard. I was hoping your script would also work for this. When I tried building and running, I got:

```
Could not open device: LIBUSB_ERROR_ACCESS
Logitech K290 not found on this system
```

So then looking over the code, I saw the `idVendor` and `idProduct` appear in `99-k290-config.rules` and `k290_fnkeyctl.cpp`. I was then hoping I could change those values that appear for the K760 (`19ff`, `0239`), that the script would work for me. After trying that, I saw:

```
Successfully set function keys to regular behaviour
```

Unfortunately, the function keys behavior did not change: they still act as specialized/media keys unless I hold Fn. I wonder if you have any insights into what I might try next?

**Comment by milgner** [comment]
This is indeed a tricky situation. Does the Windows software have an option to reverse the behaviour with a checkbox? In that case we might try capturing the USB data and reverse engineer it, too.
Unfortunately I don't have access to any other Logitech Keyboard so that I could compare their behaviour...

**Comment by allantokuda** [comment]
Unfortunately I decided to get a refund on the keyboard, so I won't be able to answer about the Windows software. For the record, the tactile experience on the K760 was excellent. But I decided using this keyboard on Linux with no vendor-supported software is probably too much work for me at this moment in my life. Not only because I want to use the F-keys, but also because pairing via bluetooth in the first place was a challenge. But I have no doubt a clever programmer could work out a good solution to these issues.
